src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/tests/Action.test.js

```diff
@@ -1,46 +1,63 @@
 // @flow
-import {render, shallow} from 'enzyme';
+import {fireEvent, render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import React from 'react';
 import Action from '../Action';
 
 test('The component should render', () => {
     const onClick = jest.fn();
     const afterAction = jest.fn();
-    const action = render(<Action afterAction={afterAction} onClick={onClick} value="my-option">My action</Action>);
-    expect(action).toMatchSnapshot();
+    const {container} = render(
+        <Action afterAction={afterAction} onClick={onClick} value="my-option">My action</Action>
+    );
+
+    expect(container).toMatchSnapshot();
 });
 
 test('The component should call the callbacks after a click', () => {
     const onClick = jest.fn();
     const afterAction = jest.fn();
-    const action = shallow(<Action afterAction={afterAction} onClick={onClick} value="my-option">My action</Action>);
-    action.find('button').simulate('click');
+    render(<Action afterAction={afterAction} onClick={onClick} value="my-option">My action</Action>);
+
+    const button = screen.queryByText('My action');
+    // eslint-disable-next-line testing-library/prefer-user-event
```

Comment:
can you add why is user-event here not possible?

Reply:
I have no idea. But when doing the same thing with `userEvent` the test just fails.

Reply:
is it possible that you forgot to `await` the interaction. i forgot about that a few times 🙈